packages/manager/apps/telecom/src/components/telecom/telephony/group/number/number.html

```diff
@@ -1,188 +1,209 @@
-<form
-    class="telephony-number"
-    name="telephonyNumber"
-    data-ng-submit="$ctrl.saveNumber()"
-    novalidate
->
-    <div
-        class="voip-plan"
-        data-ng-if="!$ctrl.loading.init"
-        data-tuc-jsplumb="$ctrl.jsPlumbInstanceOptions"
-        data-tuc-jsplumb-instance="$ctrl.jsplumbInstance"
+<div id="groupNumber">
+    <div class="group-number-switch">
+        <div data-translate="telephony_number_action_layout"></div>
+        <span
+            class="oui-icon oui-icon-ellipsis-vertical"
```

Comment:
Not sure that icons are easy to understand. Maybe we should replace by the text ?

Reply:
No